command/clistate/state.go

```diff
@@ -152,6 +152,8 @@ func (l *locker) Unlock(parentErr error) error {
 
 		if parentErr != nil {
 			parentErr = multierror.Append(parentErr, err)
+		} else {
+			return err
 		}
```

Comment:
```suggestion
		parentErr = multierror.Append(parentErr, err)
```
`multierror.Append` will create a new `*multierror.Error` if `parentErr` is nil so this is always safe.